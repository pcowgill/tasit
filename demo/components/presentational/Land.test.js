import React from "react";
import { shallow } from "enzyme";
import Land from "./Land";

describe("Land", () => {
  it("renders the component", async () => {
    const land = {
      id: -1,
      name: "Not found",
      img: require("../../assets/images/icon.png"),
      priceMana: 0,
      priceUsd: 0,
    };
    expect(shallow(<Land land={land} />)).toMatchSnapshot();
  });
});
