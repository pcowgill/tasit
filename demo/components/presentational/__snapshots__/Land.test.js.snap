// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Land renders the component 1`] = `
<Component
  style={
    Object {
      "width": 450,
    }
  }
>
  <Component
    source={1}
  />
  <Component>
    Not found
  </Component>
  <Component>
    0
     MANA (~$
    0
    )
  </Component>
</Component>
`;
